src/styles/block/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  height: 60px;
+  margin-left: 20px;
+  align-items: center;
+  color: $primary-color;
```

Comment:
It seems like you are using a variable `$primary-color` for the color property, but there is no definition of this variable in the provided code. Ensure that the variable is defined in the `variables.scss` file or provide a fallback value.